mm2src/mm2_main/src/lp_ordermatch.rs

```diff
@@ -331,6 +332,17 @@ async fn process_orders_keep_alive(
     Ok(())
 }
 
+fn process_maker_order_created(
+    ctx: &MmArc,
+    from_pubkey: String,
+    created_msg: new_protocol::MakerOrderCreated,
+) -> OrderbookP2PHandlerResult {
+    let order: OrderbookItem = (created_msg, from_pubkey).into();
+    insert_or_update_order(ctx, order);
+
+    Ok(())
```

Comment:
This seems like a error-free function. We can remove the return type.

Reply:
https://github.com/KomodoPlatform/komodo-defi-framework/pull/2232/commits/158252c6d60ddf24d7dea72abe43f8a5c7976a99